go.mod

```diff
@@ -160,3 +160,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/moby/buildkit => ../buildkit
```

Comment:
Looks to be an oversight :sweat_smile:

Reply:
It's intentional until #4213 merges - I'll replace with a pointer to my fork though.